docs/apps/snakemake.md

```diff
@@ -0,0 +1,58 @@
+---
+tags:
+  - Free
+---
+
+# Snakemake
+
+Snakemake is a python-based scientific workflow management system for creating scalable,
+portable, and reproducible workflows. It is one of the popular workflow managers within
+the bioinformatics community.
+
+[TOC]
+
+## Available 
+
+Versions available at CSC:
+
+* Puhti: 7.15.2, 7.17.1
+
+## License
+
+Snakemake is released under the
+[MIT License](https://snakemake.readthedocs.io/en/stable/project_info/license.html).
+
+## Usage
+
+Snakemake is activated by loading `snakemake` module as below:
+
+```bash
+module load snakemake
+```
+
+Example of loading `nextflow` module with a specific version:
+
+```bash
+module load snakemake/7.17.1
+```
+
+For usage help, use command:
+
+```bash
+snakemake --help
+```
+
+## References
+
+If you use Snakemake in your work, please cite:
+
+Mölder, F., Jablonski, K.P., Letcher, B., Hall, M.B., Tomkins-Tinch, C.H., Sochat, V.,
+Forster, J., Lee, S., Twardziok, S.O., Kanitz, A., Wilm, A., Holtgrewe, M., Rahmann, S.,
+Nahnsen, S., Köster, J. Sustainable data analysis with Snakemake. F1000Research 2021,
+<https://doi.org/10.12688/f1000research.29032.1>
+
+## More information
+
+* [Snakemake official documentation](https://snakemake.readthedocs.io/en/stable/index.html)
+* [Running Snakemake on Puhti](../support/tutorials/snakemake-puhti.md)
```

Comment:
To make it more visible to users visiting this page, this link could be moved to the end of the usage section? 
Something like 
"Please refer to the [Running Snakemake on Puhti](../support/tutorials/snakemake-puhti.md) tutorial for an example workflow." 
?

Reply:
Modified accordingly